src/config.ts

```diff
@@ -1,4 +1,5 @@
-import { Layer } from './oasis-indexer/api'
+// eslint-disable-next-line no-restricted-imports
+import { Layer } from './oasis-indexer/generated/api' // We get this from the generated code to avoid circular imports
```

Comment:
Should also revert e6657af7a1e65bf9374a0a738fcf9974cff5e892

Reply:
> Should also revert e6657af7a1e65bf9374a0a738fcf9974cff5e892

See https://github.com/oasisprotocol/explorer/pull/227